chi2/chinotype.py

```diff
@@ -626,7 +704,7 @@ def chi2_output(self, db):
         if len(self.filter) > 0:
             log.info('Filters: {0}'.format(self.filter))
             if 'ALL' in self.filter: self.filter.remove('ALL')
-            cols, rows = do_log_sql(db, filterStr, self.filter)
+            cols, rows = do_log_sql(db, filterStr)
```

Comment:
This is not necessary. This was broken by the above "bug fix" of a non bug. Filtering could be improved though, and at the minimum the original code obviously lacks documentation.